AfrikaBurn/MapViewController.swift

```diff
@@ -0,0 +1,339 @@
+//
+//  MapViewController.swift
+//  AfrikaBurn
+//
+//  Created by Daniel Galasko on 2017/03/04.
+//  Copyright © 2017 AfrikaBurn. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import RealmSwift
+
+struct Resources {
+    static let mapOverlayImage: UIImage = {
+        let path = Bundle.main.path(forResource: "AB2017_Layout_RoadsLabel_20170308", ofType: "png")!
+        return UIImage(contentsOfFile: path)!
+    }()
+}
+
+struct BurnMap {
+    static let boundingMapRect: MKMapRect = MKMapRect(origin: MKMapPoint(x: 148937508.42330855, y: 159711548.54660633), size: MKMapSize(width: 9388.6817751824855, height: 7286.825931340456))
+}
+
+class MapViewController: UIViewController {
+    
+    var selectedElement : AfrikaBurnElement? = nil
+    @IBOutlet weak var mapView: MKMapView!
+    let persistentStore = PersistentStore()
+    let locationManager = CLLocationManager()
+    lazy var allElements: Results<AfrikaBurnElement> = self.persistentStore.elements()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        locationManager.requestWhenInUseAuthorization()
```

Comment:
we should also show an alert if the user denies authorization so they can at least open settings and enable it later